index.js

```diff
@@ -63,6 +66,10 @@ function Cabal (storage, key, opts) {
 }
 
 inherits(Cabal, events.EventEmitter)
+Cabal.prototype.getProtocolVersion = function (cb) {
+  if (!cb) cb = noop
+  cb(PROTOCOL_VERSION)
```

Comment:
This is called synchronously (unless you use something else to make it async), so should probably do `process.nextTick(cb, PROTOCOL_VERSION)`.